app/src/renderer/components/staking/DelegationModal.vue

```diff
@@ -122,95 +131,93 @@ export default {
       type: String,
       required: true
     },
+    validator: {
+      type: Object,
+      required: true
+    },
     denom: {
       type: String,
       required: true
     }
   },
   data: () => ({
-    amount: 0,
-    selectedIndex: 0,
+    amount: ``,
```

Comment:
amount `string`?  why ?

Reply:
changed it to `null`. needs to be different from `0` to show placeholder.